src/utils/FontManager.js

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2019 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+ * Based on...
+ * ChromaCheck 1.16
+ * author Roel Nieskens, https://pixelambacht.nl
+ * MIT license
+ */
+
+let colrFontSupported = undefined;
+
+async function isColrFontSupported() {
+    if (colrFontSupported !== undefined) {
+        return colrFontSupported;
+    }
+
+    // Firefox has supported COLR fonts since version 26
+    // but doesn't support the check below with content blocking enabled.
```

Comment:
I am curious what actually goes wrong with the content blocking, but it doesn't seem worth digging into right now...